views/secretWord.ejs

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi"
+      crossorigin="anonymous"
+    />
+    <title>Jobs List</title>
+  </head>
+  <body>
+    <%- include("partials/head.ejs") %>
+    <%- include("partials/header.ejs") %>
+    <p>The secret word is: <%= secretWord %></p>
+    <p>Would you like to change it?</p>
+    <form method="POST">
+        <label name="secretWord"></label>
+        <input name="secretWord" /><br />
```

Comment:
Add this field `<input type="hidden" name="_csrf" value="<%= _csrf %>">` after this line. It's required for csrf validator. 
See it in the documentation: 
<img width="611" alt="Screenshot 2024-01-29 at 22 44 02" src="https://github.com/rebs1203/groceries-ejs/assets/377074/1c62727e-6711-458c-98b4-6296ba1437bf">